cli/flox-rust-sdk/src/providers/services.rs

```diff
@@ -624,7 +620,7 @@ mod tests {
 
             let max_tries = 5;
             for backoff in 1..max_tries {
-                println!("waiting for socket to exist");
+                debug!("waiting for socket to exist");
                 thread::sleep(Duration::from_millis(100 * backoff));
```

Comment:
unrelated, but i think we should include https://docs.rs/tracing-test/latest/tracing_test/ to have a chance to get some idea what's going on without immediately going into using a debugger